src/sections/Planning/Planning.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PlanningCard from './PlanningCard';
+var planningOneImage = require("../../assets/planning-1.png");
+var planningTwoImage = require("../../assets/planning-2.png");
+var planningThreeImage = require("../../assets/planning-3.png");
+var planningFourImage = require("../../assets/planning-4.png");
+import constants from '../../utils/constants';
+import "./Planning.scss";
+import Heading from '../../components/Heading/Heading';
+
+const Planning = () => {
+  const {planningOne,planningTwo,planningThree,planningFour} = constants.planning;
+
+  return (
+    <div className='planning section'>
+       <Heading className='planning__heading'>How do we do it?</Heading>
+       <div className='planning-cards'>
+          <PlanningCard image={planningOneImage} title={planningOne.title} description={planningOne.description} 
```

Comment:
same as above comment. can use map function if planning objects have all the data inside them

Reply:
Done